docs/source/index.rst

```diff
@@ -34,6 +34,7 @@ NVIDIA NeMo User Guide
    asr/speaker_recognition/intro
    asr/speaker_diarization/intro
    asr/ssl/intro
+   asr/speech_intent_slot/intro
```

Comment:
asr/spoken_language_understanding/intro at this level.
It should have a link pointing to asr/spoken_language_understanding/speech_intent_slow/intro.